meson.build

```diff
@@ -1012,23 +1012,35 @@ else
                 clang_supports_bpf = false
         endif
 
-        if not meson.is_cross_build() and clang_found
-                llvm_strip_bin = run_command(clang, '--print-prog-name', 'llvm-strip',
-                                             check : true).stdout().strip()
-        else
-                llvm_strip_bin = 'llvm-strip'
-        endif
-        llvm_strip = find_program(llvm_strip_bin, required : bpf_framework_required, version : '>= 10.0.0')
-
         # Debian installs this in /usr/sbin/ which is not in $PATH.
         # We check for 'bpftool' first, honouring $PATH, and in /usr/sbin/ for Debian.
         # We use 'bpftool gen' subcommand, it was added by 985ead416df39d6fe8e89580cc1db6aa273e0175 (v5.6).
         bpftool = find_program('bpftool',
                                '/usr/sbin/bpftool',
-                               required : bpf_framework_required,
-                               version : '>= 5.6')
+                               required : false,
+                               version : '>= 5.13.0')
+
+        if bpftool.found()
+                bpftool_strip = true
+        else
+                bpftool_strip = false
+                bpftool = find_program('bpftool',
+                                       '/usr/sbin/bpftool',
+                                       required : bpf_framework_required,
+                                       version : '>= 5.6.0')
```

Comment:
I changed this to the full 3 part version string since that seems to match the format output from `bpftool`:
```
$ ./bpftool --version
./bpftool v5.16.0-c446fdacb10d
features: libbpf_strict
```